WPMedia/fusion-news-theme-blocks
# TMEDIA-458 - Subscriptions Single Column Layout

## Description

Add a new layout "Single Column" to the Identity block

## Jira Ticket
- [TMEDIA-458](https://arcpublishing.atlassian.net/browse/TMEDIA-458)

## Acceptance Criteria
* Create a new layout called Single Column with three sections:
* full width nav
* main single column w/max width constraints
* full width footer
* Part of the Identity block

## Test Steps

1. Checkout this branch `git checkout TMEDIA-458-identiy-single-column-layout`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block`
3. In PageBuilder editor validate the new layout is available in the layout dropdown

Check Chromatic for storybook outputs - https://www.chromatic.com/build?appId=5f91e4721ccaba0022a10782&number=817

## Effect Of Changes

<img width="285" alt="TMEDIA-458-layout-options" src="https://user-images.githubusercontent.com/868127/130222686-a5801ffe-c4e4-4e93-b76b-47c99b3ddac5.png">

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
